imagepicker/src/main/java/com/esafirm/imagepicker/adapter/ImagePickerAdapter.java

```diff
@@ -60,6 +60,10 @@ public void onBindViewHolder(ImageViewHolder viewHolder, int position) {
                 ? View.VISIBLE
                 : View.GONE);
 
+        viewHolder.videoIndicator.setVisibility(ImagePickerUtils.isVideoFormat(image)
```

Comment:
Same as the previous one. This could be merged with gif indicator since it mutually exclusive.